transformations/gender_neutral_rewrite/transformation.py

```diff
@@ -0,0 +1,425 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import torch
+from string import punctuation
+
+from transformations.gender_neutral_rewrite.myconstants import *
+
+# direct replacement mapping
+SIMPLE_REPLACE = EASY_PRONOUNS
+SIMPLE_REPLACE.update(GENDERED_TERMS)
+
+# load SpaCy's "en_core_web_sm" model
+# English multi-task CNN trained on OntoNotes
+# Assigns context-specific token vectors, POS tags, dependency parse and named entities
+# https://spacy.io/models/en
+import en_core_web_sm
+
+nlp = en_core_web_sm.load()
```

Comment:
Can you load this using the global spacy load?
```python
from initialize import spacy_nlp
import spacy
self.nlp = spacy_nlp if spacy_nlp else spacy.load("en_core_web_sm")
```

Reply:
Done. Thanks for the suggestion.